### PR TITLE
Added vulnerability identifier

### PR DESCRIPTION
**Pull request by Cheshiriks**
- added field Identifier in vulnerability
- added field Identifier on vulnerability view
- added check that a vulnerability with this ID does not yet exist

### CODE REVIEW

**Comment by akuleshov7** [comment]
where is a field for CVE? Can you pls add a full screenshot?

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/FossGraphView.kt [hunk 1]
```diff
@@ -352,7 +352,26 @@ val fossGraph: FC<FossGraphViewProps> = FC { props ->
...
-
+                    div {
+                        className = ClassName("card-header py-3")
+                        div {
+                            className = ClassName("row")
+                            h6 {
+                                className = ClassName("m-0 font-weight-bold text-primary")
+                                style = jso {
+                                    display = Display.flex
+                                    alignItems = AlignItems.center
```
Comment (sanyavertolet) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/FossGraphView.kt:363]: You can use classnames for this purpose

**Review by sanyavertolet — APPROVED** [review]

**Comment by Cheshiriks** [comment]
> where is a field for CVE? Can you pls add a full screenshot?

Added

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/FossGraphView.kt (hunk 1 above)
Comment (Cheshiriks) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/FossGraphView.kt]: Fixed

**Comment by sanyavertolet** [comment]
Why is input with CVE displayed as `success` instead of `danger`?

**Pull request merged**